styles.css

```diff
@@ -1,12 +1,12 @@
 .container{
-    width: 70%;
+    width: 65%;
```

Comment:
this is okay, but you can exclude the navbar from this container, just to make you web UX a little bit better,